app/views/components/pageflow/admin/add_membership_button.rb

```diff
@@ -56,7 +62,7 @@ def entry_membership_possible_for_creator_and(user, parent)
         when User
           PotentialMemberships.creatable_by(user).entries_for_user(parent).any?
         when Entry
-          PotentialMemberships.creatable_by(user).users_for_entry(parent).any?
+          PotentialMemberships.creatable_by(user).users_for_entry(parent).without(user).any?
```

Comment:
This hides the button on the entry admin page if there is only one user in the account. But if the user visits the user admin page of their own user record, they can still add entries there. I think this is inconsistent.

On the other hand, if there are many users in the account and i want to assign users to entries to make clear who is responsible for what, why can't I add myself?

Also if all users of an account have been added to an entry, there is no need to display a message that you need to invite even more users.

I think, the important point is that "member management" does not make sense if there is only user in the account. So if I am alone in my account, I should not see the "Add user" button on the entry page. Instead there should be a text explaining what the membership concept is for.

In the context of Dedicated (where there are no quotas) or if my quota is not exhausted, this text can tell me to first invite users into my account before I can start managing memberships.

If my quota is exhausted, it can tell me to first go upgrade to use the feature. Since upgrading is only a concept in Hosted Pageflow, the partial inside of Pageflow can only include a generic message stating that the quota is exhausted. This can then be overridden in Hosted Pageflow to include a link to the upgrade page.

Reply:
So now it does not show add user button if users in account are less than 2.
In other cases when membership is not possible button gets disabled.

In both cases It checks if quota is exhausted and display quota exhausted message otherwise it displays invite more users message.

template pageflow/admin/entries/cannot_add_user can be overriden by host application